### DISCUSSION TITLE
Wrong Version on the bottom right

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#15057] [Help Wanted!]
### Deployment Type

Self-hosted

### NetBox Version

v3.7.2

### Python Version

3.11

### Steps to Reproduce

Open Netbox with a Version >3.7.0

### Expected Behavior

The Verison (currently v3.7.2) should be display

### Observed Behavior

I noticed the Version number on the bottom right didn't change after v3.7.0. I'm currently running v3.7.2 but it still shows v3.7.0

**Comment by candlerb** [comment]
The simple answer is you're not running 3.7.2, even if you think you are. Something's gone wrong with your upgrade process.

For a regular server installation, there are [two different ways](https://docs.netbox.dev/en/stable/installation/upgrading/) of doing the upgrade: one if you originally installed from git, and one if you originally installed from a tarball. You have to use the one which matches your original installation method.

The git approach is simpler, basically just `git pull` followed up `./upgrade.sh`. The tarball approach involves untarring a new version into a new directory, modifying a symlink, and copying various files from the old directory to the new one.

Either way, you must remember to restart netbox.  And if you see any errors during `./upgrade.sh` you can't just ignore them.

If you're running under docker or kubernetes, then you're not running the right container image version.

**Reply by PaulR282** [reply]
I'm running it as a docker container. I install it with the v3.7-2.8.0 tag. It should still be the v3.7.2 version because I cannot reproduce bugs that have been fixed with that version. Is it a problem with netbox-docker?

**Reply by candlerb** [reply]
I'm pretty sure you're running the wrong image. You didn't say where you got it from (could be docker hub, quay.io or Github Container Registry)

However, looking on [docker hub](https://hub.docker.com/r/netboxcommunity/netbox/tags) I see that there is a tag [v3.7.2-2.8.0](https://hub.docker.com/layers/netboxcommunity/netbox/v3.7.2-2.8.0/images/sha256-8792b955bdc8c80927052dcce297e9ce4a6700b63ffc14e0ba058bd8fa78f401?context=explore) which sounds like what you should be using.

EDIT: actually v3.7-2.8.0 and v3.7.2-2.8.0 both have the same digest (8792b955bdc8), so I guess that the v3.7-2.8.0 tag has been slipped. But if you fetched the image with that tag previously, you might not have got the new one.

In the netbox-docker git repository, 2.8.0 was tagged on Tue Jan 2 08:57:00 2024 +0100. But Netbox v3.7.2 wasn't tagged until Mon Feb 5 14:10:13 2024 -0500. As far as I can tell, the netdox-docker scripts by default build the 'master' branch.

**Reply by PaulR282** [reply]
Yes, I am getting the image from the ducker hub. I used the v3.X.2-X.X tag all the time and it worked. When using v3.7.2-2.8.0 the problem still exists. So I guess it's a problem with netbox-docker. I will open an issue there.
